doc/CMakeLists.txt

```diff
@@ -75,34 +74,22 @@ function(gen_manpage sourcefile man_nr)
             COMMENT "Using pre-built ${dst_prebuilt}"
             )
     else()
-        # if the documentation is built from scratch, add an intermediate step for
-        # the xml-based docbook format.
-        # add_dependencies("doc_man_${sourcefile}" "doc_man_${sourcefile}_xml")
-        # add_custom_target("doc_man_${sourcefile}_xml" ALL DEPENDS ${docbookxml})
+        # create a config file such that we can tell a2x to call
+        # our script patching the man page xml instead of running xmllint
+        set(custom_asciidoc_cfg "${CMAKE_CURRENT_BINARY_DIR}/a2x-patched-dont-install.conf")
+        file(WRITE ${custom_asciidoc_cfg}
+        "XMLLINT = '${CMAKE_CURRENT_SOURCE_DIR}/patch-manpage-xml.py'")
         add_custom_command(
-            OUTPUT ${dst} ${docbookxml}
-            # first create the docbook xml:
-            COMMAND ${ASCIIDOC}
-                    -o ${docbookxml}
-                    -b docbook45
+            OUTPUT ${dst}
+            COMMAND ${ASCIIDOC_A2X}
+                    --verbose
```

Comment:
Do we still want this or was this just for debugging purposes?

Reply:
I'd like to keep it because otherwise, a2x shows too little information.

Reply:
Alright! Just wanted to make sure this is intended :)